magellan-sample-advanced/src/main/java/com/wealthfront/magellan/sample/advanced/SampleApplication.kt

```diff
@@ -12,6 +19,19 @@ class SampleApplication : Application() {
     appComponent = DaggerAppComponent.builder()
       .appModule(AppModule())
       .build()
+
+    Magellan.navigationRequestHandler = object: NavigationRequestHandler {
+      override fun onNavigationRequested(
+        navigationDelegate: NavigationDelegate,
+        navigable: NavigableCompat
+      ): Boolean {
+        if (navigable is SuggestExhibitJourney) {
```

Comment:
arbitrary example: this version of the app shouldn't be allowed to see SuggestExhibitJourney